geonode_mapstore_client/client/js/apps/gn-components.js

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2022, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { connect } from 'react-redux';
+import main from '@mapstore/framework/components/app/main';
+import ComponentsRoute from '@js/routes/Components';
+import MainLoader from '@js/components/MainLoader';
+import Router, { withRoutes } from '@js/components/Router';
+import security from '@mapstore/framework/reducers/security';
+import {
+    getEndpoints,
+    getConfiguration,
+    getAccountInfo
+} from '@js/api/geonode/v2';
+import {
+    setupConfiguration,
+    initializeApp,
+    storeEpicsCache,
+    getPluginsConfiguration,
+    getPluginsConfigOverride
+} from '@js/utils/AppUtils';
+import pluginsDefinition from '@js/plugins/index';
+import StandardApp from '@mapstore/framework/components/app/StandardApp';
+import withExtensions from '@mapstore/framework/components/app/withExtensions';
+import gnsettings from '@js/reducers/gnsettings';
+import { updateGeoNodeSettings } from '@js/actions/gnsettings';
+
+const requires = {};
+
+initializeApp();
+
+const DEFAULT_LOCALE = {};
+const ConnectedRouter = connect((state) => ({
+    locale: state?.locale || DEFAULT_LOCALE
+}))(Router);
+
+
+const routes = [{
```

Comment:
Here, I think we should add the routes and import from the new [routes utils](https://github.com/GeoNode/geonode-mapstore-client/blob/master/geonode_mapstore_client/client/js/utils/AppRoutesUtils.js) to make the workflow consistent